database/helper.js

```diff
@@ -34,12 +34,39 @@ module.exports.addSearchResult = (obj) => {
     searchQueryId, availableListings, scoringRules,
   } = obj;
   const results = availableListings.map((result) => {
-    const resultText = `INSERT INTO listings (search_id, listing_id, scoring_rules) 
-      VALUES (${searchQueryId}, '${result.listingId}', '${scoringRules}'::json) 
-      ON CONFLICT DO NOTHING`;
+    const resultText = `INSERT INTO listings (search_id, listing_id, scoring_rules) VALUES (${searchQueryId}, '${result.listingId}', '${scoringRules}'::json) ON CONFLICT DO NOTHING`;
```

Comment:
I'm a little confused here -- this following query is in a different DB than the previous ones, correct? Can you clarify the purpose of this resultText query this for me?

Reply:
This query is for consumption by Search Service as weights/coefficients to apply to the Search sorting. It's a little confusing since I snuck in a "combine all into one line" cosmetic change into this pull request. I should have done those all in a "minor fixes" pull request.